docs/pages/prototyping-utilities.md

```diff
@@ -554,6 +560,12 @@ Generate spaces around the content with these easy to use padding classes.
 <div class="padding-right-3"></div>
 ```
 
+<div class="callout warning">
+  <p>
+    <code>padding-horizontal</code> and <code>padding-vertical</code> classes have been depreciated and been replaced with <code>padding-x</code> and <code>padding-y</code> classes to keep our code consistent with <a href="xy-grid.html"><strong>XY Grid</strong></a> 
```

Comment:
Deprecated means they are still around but will be removed, lets change this simply to say they have been replaced.